Tests/HttpUnitTests/HttpContentTest.cs

```diff
@@ -0,0 +1,131 @@
+﻿//
+// Copyright (c) .NET Foundation and Contributors
+// Portions Copyright (c) Microsoft Corporation.  All rights reserved.
+// See LICENSE file in the project root for full license information.
+//
+
+using nanoFramework.TestFramework;
+using System;
+using System.IO;
+using System.Net.Http;
+using System.Reflection;
+
+namespace HttpUnitTests
+{
+    [TestClass]
+    public class HttpContentTest
+    {
+        [TestMethod]
+        public void Dispose_BufferContentThenDisposeContent_BufferedStreamGetsDisposed()
+        {
+            Assert.SkipTest("Skipping test because of issue in Test Framework");
+
+            // TODO
+            // test is being reported as failing, despite it's successful, issue lyes in the TestFramework
```

Comment:
what is the problem? Can you open an issue with it so someone can fix it later?

Reply:
Need to debug deeper. Can't say exactly what it is but it's coming from there.